### DISCUSSION TITLE
V0.2

### DISCUSSION THREAD

**Discussion by PonteIneptique** [HTR-United/schema#7] [General]
I have selected four issues that I believe we need to treat quite quickly (before 15th of April ?)
What do you think @alix-tz

**Comment by alix-tz** [comment]
I can't see the selection of issues 🤔

**Comment by alix-tz** [comment]
Ah, it's all good : https://github.com/HTR-United/schema/milestone/1

**Comment by PonteIneptique** [comment]
They are marked with milestone v. 0.2

Le mar. 22 mars 2022 à 6:50 PM, Alix Chagué ***@***.***> a
écrit :
